src/mappings/position.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  Position as PositionContract,
+  PositionOpened,
+  PositionClosed,
+  FeatureCalled,
+} from '../../generated/templates/Position/Position'
+import { EntryPoint, Position, PositionDailyData } from '../../generated/schema'
+import { ADDRESS_ZERO, BIGINT_ONE, BIGINT_ZERO } from '../utils/constants'
+import { Address, Bytes, ethereum } from '@graphprotocol/graph-ts'
+import { ENTRY_POINT } from '../utils/config'
+
+export function handlePositionOpened(event: PositionOpened): void {
+  const position = Position.load(event.address)!
+  position.openedAt = event.block.timestamp
+  position.txCount = position.txCount.plus(BIGINT_ONE)
+  position.totalAllocated = event.params.pushed
+  position.asset = event.params.asset
+  // Read data from contract
+  const positionContract = PositionContract.bind(event.address)
+  position.pricePerShare = positionContract.pricePerShare()
+  position.isOutdated = positionContract.isOutdated()
+
+  position.save()
+}
+
+export function handlePositionClosed(event: PositionClosed): void {
+  const position = Position.load(event.address)!
+  position.totalAllocated = event.params.pulled
+  position.openedAt = BIGINT_ZERO
+  position.txCount = BIGINT_ZERO
+  position.asset = ADDRESS_ZERO
+  position.pricePerShare = BIGINT_ZERO
```

Comment:
I think we could update the `isOutdated` field here too. A closed position can be reused (as a way to save gas, avoiding deploying another proxy) so with this flag may trigger an update before re-opening it.

Reply:
With triggering I mean, bundle [upgrade, open] txs

Reply:
Good point. I will look into this.

Reply:
So here is the deal with this flag. Say we fetch latest status of strategy at the time of close and set it in `isOutdated`. Status can be updated between it is closed and opened and there is no way on subgraph to determine this. Only possible solution would be to check `isOutdated()` on-chain before reusing position. 
We can still set latest at the time of close but it won't be much help IMO.

Reply:
Done

Reply:
> So here is the deal with this flag. Say we fetch latest status of strategy at the time of close and set it in `isOutdated`. Status can be updated between it is closed and opened and there is no way on subgraph to determine this. Only possible solution would be to check `isOutdated()` on-chain before reusing position. We can still set latest at the time of close but it won't be much help IMO.

Yes, indeed, let's keep it in mind. FYI @mbonfoster